orchestration/hca_orchestration/resources/config/datasets.py

```diff
@@ -0,0 +1,84 @@
+import json
+import uuid
+from datetime import datetime
+from typing import Optional
+
+from dagster import InitResourceContext, resource, Array, Field, Noneable
+
+from hca_orchestration.contrib.data_repo.data_repo_service import DataRepoService
+from hca_orchestration.models.hca_dataset import TdrDataset
+from hca_orchestration.support.dates import dataset_snapshot_formatted_date
+
+
+@resource(
+    required_resource_keys={"data_repo_service"},
+    config_schema={
+        "dataset_id": str
+    }
+)
+def passthrough_hca_dataset(init_context: InitResourceContext) -> TdrDataset:
```

Comment:
This is the previous `target_hca_dataset` method, slightly renamed in a way that is hopefully more obvious.